src/wowman/core.clj

```diff
@@ -579,6 +612,81 @@
 
       (update-installed-addon-list! expanded-installed-addon-list))))
 
+;; catalog db handling
+
+(defn-spec db-init nil?
+  []
+  (debug "creating 'catalog' table")
+  (jdbc/execute! (get-state :db) [(slurp "resources/table--catalog.sql")])
+  (debug "creating 'category' table")
+  (jdbc/execute! (get-state :db) [(slurp "resources/table--category.sql")])
+
+  (swap! state update-in [:cleanup] conj (fn []
+                                           (try
+                                             (db-query "shutdown")
+                                             (catch org.h2.jdbc.JdbcSQLNonTransientConnectionException e
+                                               ;; "Database is already closed" (it's not)
+                                               (debug (str e))))))
+  nil)
+
+(defn db-catalog-loaded?
+  []
+  (> (get-db :catalog-size) 0))
+
+;; slow and stupid way of loading addons into a normalised db schema
```

Comment:
pretty harsh. it's one of two or three possible ways of quickly bulk loading items. now that we have a baseline we can try different ways of improving it. remove comment